source-manager/src/main/java/com/jointrivial/sourcemanager/nordigen/model/view/KeyOrganisationViewModel.java

```diff
@@ -0,0 +1,28 @@
+package com.jointrivial.sourcemanager.nordigen.model.view;
+
+public class KeyOrganisationViewModel {
+
+    private String institutionId;
```

Comment:
change field name to organizationName

Reply:
There is just no institutionId. This has to be orgName, no? And if anything, just consistency.